docs/spelling_wordlist.txt

```diff
@@ -14,6 +15,7 @@ cli
 CLI
 clusterhq
 Copr
+cURL
```

Comment:
There are exceptions but I think use this file for words you think should be in the dictionary, else wrap in double backticks.